src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react';
+import { Todo } from '../TodoInfo';
+import usersFormServer from '../../api/users';
+import { getUserById } from '../../services/getUser';
+
+type Props = {
+  onSubmit: (todo: Todo) => void;
+};
+
+export const TodoForm: React.FC<Props> = ({ onSubmit }) => {
+  const [title, setTitle] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [userNameError, setUserNameError] = useState(false);
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+
+    setHasTitleError(!title);
+    setUserNameError(!userId);
+
+    if (!title || !userId) {
+      return;
+    }
+
+    onSubmit({
+      id: 0,
```

Comment:
why do you need to pass `id: 0` and then change it? Just set the correct `id` here or don't pass it